docs/principles/cloud-native.md

```diff
@@ -1,9 +1,7 @@
 ## Docker, Kubernetes and Cloud Native
 
-True portability, though, comes from using industry standard [Docker containers](https://www.docker.com/resources/what-container) for packaging and deploying our workloads (web apps, microservices, legacy apps etc). They can be deployed to any orchestrator (Kubernetes, DC/OS, Mesos, Docker Swarm), but 2 years ago [Kubernetes](https://kubernetes.io/docs/concepts/overview/what-is-kubernetes/) won the battle and is now used almost universally at organisations across the world. All the major cloud providers have managed Kubernetes offerings ([AWS EKS](https://aws.amazon.com/eks/), [GCP GKE](https://cloud.google.com/kubernetes-engine/) and [Azure AKS](https://azure.microsoft.com/en-us/services/kubernetes-service/)), which are free to use (you only pay for the VMs that form the nodes in the cluster).
+True portability comes from using industry standard [Docker containers](https://www.docker.com/resources/what-container) for packaging and deploying our workloads (web apps, microservices, legacy apps etc). They can be deployed to any orchestrator (Kubernetes, DC/OS, Mesos, Docker Swarm), but 2 years ago [Kubernetes](https://kubernetes.io/docs/concepts/overview/what-is-kubernetes/) won the battle and is now used almost universally at organisations across the world. All the major cloud providers have managed Kubernetes offerings ([AWS EKS](https://aws.amazon.com/eks/), [GCP GKE](https://cloud.google.com/kubernetes-engine/) and [Azure AKS](https://azure.microsoft.com/en-us/services/kubernetes-service/)), which are free to use (you only pay for the VMs that form the nodes in the cluster).
```

Comment:
line-breaking after full-stops or commas makes it a lot easier to see what's changed in future iterations